svd-parser v0.14.3 fails to compile with svd-rs version v0.14.2 and v0.14.3

Hi,

I'm seeing a problem where v0.14.3 of the svd-parser fails to compile:

    error[E0425]: cannot find function `expand` in module `cluster`
       --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/svd-parser-0.14.3/src/expand.rs:322:31
        |
    322 |             for c in cluster::expand(&info, &dim) {
        |                               ^^^^^^ not found in `cluster`
        |
    help: consider importing this function through its public re-export
        |
    3   + use crate::expand;
        |
    help: if you import `expand`, refer to it directly
        |
    322 -             for c in cluster::expand(&info, &dim) {
    322 +             for c in expand(&info, &dim) {
        |

(from https://github.com/probe-rs/probe-rs/pull/1897)



These are the dependencies:
```
 svd-parser v0.14.3
 585   │ │   ├── anyhow v1.0.75
 586   │ │   ├── roxmltree v0.18.0
 587   │ │   │   └── xmlparser v0.13.5
 588   │ │   ├── svd-rs v0.14.2
 589   │ │   │   ├── once_cell v1.18.0
 590   │ │   │   ├── regex v1.10.1 (*)
 591   │ │   │   └── thiserror v1.0.49 (*)
 592   │ │   └── thiserror v1.0.49 (*)
```

After updating `svd-rs` to version v0.14.4 it compiles again.

This seems to indicate that `svd-parser` needs to explicitly depend on the newer version of `svd-rs`.

--- a/svd-parser/CHANGELOG.md
+++ b/svd-parser/CHANGELOG.md
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+## [v0.14.4] - 2023-11-15
+
+- Bump svd-rs dependency to 0.14.4 or higher.
+
 ## [v0.14.3] - 2023-11-15
 
 - Correctly place `expand_properties` under `expand` feature
@@ -59,8 +63,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 Previous versions in common [changelog](../CHANGELOG.md).
 
-[Unreleased]: https://github.com/rust-embedded/svd/compare/svd-rs-v0.14.4...HEAD
-[v0.14.3]: https://github.com/rust-embedded/svd/compare/svd-rs-v0.14.3...svd-rs-v0.14.4
+[Unreleased]: https://github.com/rust-embedded/svd/compare/svd-parser-v0.14.4...HEAD
+[v0.14.4]: https://github.com/rust-embedded/svd/compare/svd-parser-v0.14.3...svd-parser-v0.14.4
+[v0.14.3]: https://github.com/rust-embedded/svd/compare/svd-parser-v0.14.2...svd-parser-v0.14.3
 [v0.14.2]: https://github.com/rust-embedded/svd/compare/svd-rs-v0.14.2...svd-parser-v0.14.2
 [v0.14.1]: https://github.com/rust-embedded/svd/compare/v0.14.0...svd-rs-v0.14.1
 [v0.14.0]: https://github.com/rust-embedded/svd/compare/svd-parser-v0.13.4...v0.14.0

--- a/svd-parser/Cargo.toml
+++ b/svd-parser/Cargo.toml
@@ -19,7 +19,7 @@ derive-from = ["svd-rs/derive-from"]
 expand = ["derive-from"]
 
 [dependencies]
-svd-rs = { version = "0.14.1", path = "../svd-rs" }
+svd-rs = { version = "0.14.4", path = "../svd-rs" }
 roxmltree = "0.18"
 anyhow = "1.0.58"
 thiserror = "1.0.31"

--- a/svd-parser/Cargo.toml
+++ b/svd-parser/Cargo.toml
@@ -11,7 +11,7 @@ name = "svd-parser"
 repository = "https://github.com/rust-embedded/svd"
 edition = "2021"
 rust-version = "1.58.0"
-version = "0.14.3"
+version = "0.14.4"
 readme = "README.md"
 
 [features]

